Base/usr/share/man/man6/BrickGame.md

```diff
@@ -12,4 +12,39 @@ $ BrickGame
 
 ## Description
 
-BrickGame is a classic game.
+BrickGame is a classic game. Pieces consisting of four small squares each fall from the top of the screen, begin fixed in place once they land at the very bottom or on top of other pieces. Once a piece has landed, the next piece will start falling; you can preview this piece on the right side of the screen. By filling an entire row (or line) of squares, that row will be deleted and the rows above will shift down. It is also possible to clear multiple lines at once.
+
+You can control where and how a piece falls, by moving it left or right, making it move down faster, dropping it down instantly, or rotating it left and right. There are multiple control schemes available for these six basic actions; and the space bar always serves as the instant drop key.
+- Left: Move left, Right: Move right, Up: Rotate right, Down: Move down
+- A: Move left, D: Move right, W: Rotate right, E: Rotate left, S: Move down
+- H: Move left, L: Move right, K: Rotate right
+
+The seven pieces are commonly named "T", "J", "L", "O", "S", "Z", and "I". Beware that while "J" and "L" as well as "S" and "Z" are mirrors of each other, they cannot be used interchangeably since you can only rotate pieces.
+
+```
+T:  x x x
+      x
+
+J:  x x x
+        x
+
+L:  x x x
+    x
+
+O:  x x
+    x x
+
+S:    x x
+    x x
+
+Z:  x x
+      x x
+
+I:  x x x x
+```
+
+The game will award you points for clearing lines, and the number depends both on the number of lines cleared at once, as well as your current level. The base points are 40, 100, 300, and 1200 for one, two, three, and four lines respectively, each multiplied by your current level number.
```

Comment:
no need for personification? (i.e. “you will be awarded with points”?)

Reply:
I don't think we usually avoid that.

Reply:
See e.g. final paragraph of 2048